tikv/raftstore/fsm_store.go

```diff
@@ -62,37 +62,44 @@ func (m *storeMeta) setRegion(region *metapb.Region, peer *Peer) {
 type mergeLock struct {
 }
 
-type PollContext struct {
+type GlobalContext struct {
 	cfg                  *Config
 	engine               *Engines
-	applyMsgs            *applyMsgs
-	ReadyRes             []*ReadyICPair
-	kvWB                 *WriteBatch
-	raftWB               *WriteBatch
-	syncLog              bool
+	store                *metapb.Store
 	storeMeta            *storeMeta
 	storeMetaLock        *sync.RWMutex
 	snapMgr              *SnapManager
-	pendingCount         int
-	hasReady             bool
 	router               *router
-	tickDriverCh         chan<- uint64
 	trans                Transport
-	queuedSnaps          map[uint64]struct{}
 	pdScheduler          chan<- task
-	raftLogGCScheduler   chan<- task
-	store                *metapb.Store
 	regionScheduler      chan<- task
-	splitCheckScheduler  chan<- task
 	computeHashScheduler chan<- task
+	raftLogGCScheduler   chan<- task
```

Comment:
Should we use names like `xxxTaskQueue`  and `xxxTaskHandler/Processor`  in unistore